cmd/tink-worker/container/container.go

```diff
@@ -0,0 +1,104 @@
+package container
+
+import (
+	"context"
+	"path"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/client"
+	"github.com/pkg/errors"
+)
+
+type containerConfigData struct {
+	pid            string
+	env            []string
+	containerName  string
+	volumeBindings []string
+	cmd            []string
+	imageName      string
+	registry       string
+}
+
+// create creates container and returns containerid on success else returns error.
+func create(ctx context.Context, cli client.ContainerAPIClient, configData containerConfigData, captureLogs bool) (string, error) {
+	config := &container.Config{
+		Image:        path.Join(configData.registry, configData.imageName),
+		AttachStdout: true,
+		AttachStderr: true,
+		Cmd:          configData.cmd,
+		Tty:          true,
+		Env:          configData.env,
+	}
+	if !captureLogs {
+		config.AttachStdout = true
+		config.AttachStderr = true
```

Comment:
These values are already true. do we need this? or should the values above be `false`?